src/helpers/dotEnv.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+  ENV: process.env.NODE_ENV || 'development',
```

Comment:
I think we should also do `ENV`. `process.env.NODE_ENV || process.env.ENV || 'development`